### DISCUSSION TITLE
What does ">_" mean in run command?

### DISCUSSION THREAD

**Discussion by zillionare** [community/community#26588] [Actions]
<p>I found this grammer in workflow files:</p>
<pre><code class="lang-auto">      - name: Build wheels and source tarball
        run: &gt;-
          poetry build

</code></pre>
<p>what does “&gt;_” mean?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>That’s part of YAML syntax for multiline strings. The <code>&gt;</code> says to fold lines within the following block (replace newlines with spaces), the <code>-</code> removes newlines at the end of the string.</p>
<p>See also: <a href="https://yaml-multiline.info/">https://yaml-multiline.info/</a></p>
